layerfetcher/layers.go

```diff
@@ -664,7 +664,7 @@ func (l *Logic) GetTxs(ctx context.Context, IDs []types.TransactionID) error {
 		res := <-resC
 		if res.Err != nil {
 			l.log.WithContext(ctx).With().Error("cannot find tx", log.String("hash", hash.String()), log.Err(res.Err))
-			continue
+			return res.Err
```

Comment:
yes. this would be the obvious fix. but internally we haven't decided what the expected behavior should be after this method returns error, so there are a lot more work (including test changes and other component changes) when this line is changed.

Reply:
as i remember it was only me who was against this, in order to enable pruning. but i think that it is already confirmed that there will be no pruning for transactions in genesis